environment.yml

```diff
@@ -0,0 +1,36 @@
+name: ads_venv
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - python=3.7
+  - fsleyes=0.33.1
+  - h5py=2.10.0
+  - numpy
+  - scipy
+  - scikit-learn=0.19.2
+  - scikit-image=0.14.2
+  - tabulate
+  - pandas
+  - matplotlib
+  - tensorflow=1.13.1
+  - mpld3
+  - tqdm
+  - requests
+  - pillow
+  - imageio
+  - pytest
+  - pytest-cov
+  - prettytable
+  - raven
+  - jupyter
+  - Keras
+  - Keras-Applications
+  - Keras-Preprocessing
+  - albumentations=0.3.0
+  - openpyxl
+  - h5py
```

Comment:
Ah, I just noticed that `h5py` is listed twice: Once pinned, and once unpinned. 

It would be interesting if this was the cause (probably not) but I thought I'd mention it here.